clang/test/SemaSYCL/stdtypes_kernel_type.cpp

```diff
@@ -25,38 +25,32 @@ queue q;
 int main() {
```

Comment:
Please add a brief comment at the top indicating the purpose of the test.

Reply:
Done. Thanks for catching!